test/IDE/complete_call_arg.swift

```diff
@@ -779,3 +793,26 @@ func testTypecheckedTypeExpr() {
 // TYPECHECKED_TYPEEXPR: Decl[Constructor]/CurrNominal:      ['(']{#arg1: String#}, {#arg2: _#}[')'][#MyType<_>#]; name=arg1: String, arg2: _
 // TYPECHECKED_TYPEEXPR: Decl[Constructor]/CurrNominal:      ['(']{#(intVal): Int#}[')'][#MyType<Int>#]; name=intVal: Int
 // TYPECHECKED_TYPEEXPR: End completions
+
+func testPamrameterFlags(_: Int, inoutArg: inout Int, autoclosureArg: @autoclosure () -> Int, iuoArg: Int!, variadicArg: Int...) {
+  var intVal = 1
+  testPamrameterFlags(intVal, #^ARG_PARAMFLAG_INOUT^#)
+// ARG_PARAMFLAG_INOUT: Begin completions, 1 items
+// ARG_PARAMFLAG_INOUT-DAG: Pattern/ExprSpecific: {#inoutArg: &Int#}[#inout Int#]; name=inoutArg:
+// ARG_PARAMFLAG_INOUT: End completions
+
+  testPamrameterFlags(intVal, inoutArg: &intVal, #^ARG_PARAMFLAG_AUTOCLOSURE^#)
+// ARG_PARAMFLAG_AUTOCLOSURE: Begin completions, 1 items
+// ARG_PARAMFLAG_AUTOCLOSURE-DAG: Pattern/ExprSpecific: {#autoclosureArg: Int#}[#Int#];
+// ARG_PARAMFLAG_AUTOCLOSURE: End completions
+
+  testPamrameterFlags(intVal, inoutArg: &intVal, autoclosureArg: intVal, #^ARG_PARAMFLAG_IUO^#)
+// ARG_PARAMFLAG_IUO: Begin completions, 1 items
+// ARG_PARAMFLAG_IUO-DAG: Pattern/ExprSpecific: {#iuoArg: Int?#}[#Int?#];
+// ARG_PARAMFLAG_IUO: End completions
+
+  testPamrameterFlags(intVal, inoutArg: &intVal, autoclosureArg: intVal, iuoArg: intVal, #^ARG_PARAMFLAG_VARIADIC^#)
+// ARG_PARAMFLAG_VARIADIC: Begin completions, 1 items
+// ARG_PARAMFLAG_VARIADIC-DAG: Pattern/ExprSpecific: {#variadicArg: Int...#}[#Int#];
```

Comment:
If you really don't like this. I can make it '[#Int...#]'. But I think this is OK because this specific argument itself is `Int`.

Reply:
I agree that `Int` is fine here.